next/components/ui/OfficialBoardCard/OfficialBoardCardModalContent.tsx

```diff
@@ -25,14 +25,13 @@ const OfficialBoardCardModalContent = ({ id, createdAt }: Props) => {
     select: (res) => res.data,
   })
 
-  // TODO move parsing into endpoint
   const files: FileCardProps[] =
-    data?.['Soubory-dokumentu']?.map(
-      (file: any): FileCardProps => ({
-        title: `${file.Nazev}`,
-        downloadLink: generateUrlForOfficialBoardFile(file.IdSouboru, file.Nazev),
-        format: file.Pripona?.replace(/^\./, '').toUpperCase().trim(),
-        size: file.Velikost, // It comes as formatted string already, e.g. "1,2 MB" or "126 KB"
+    (data as ParsedOfficialBoardDocumentDetail)?.files.map(
+      (file): FileCardProps => ({
+        title: file.title,
+        downloadLink: file.generatedUrl,
+        format: '',
```

Comment:
pdf ? or hardcoded elsewhere / not used ? either way, deserves a comment in code